### PR TITLE
fix: Display error messages in works to import using bibtex

### PR DESCRIPTION
**Pull request by DanielPalafox**

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[[QA] Works imported from BibTeX containing invalid URLs are not saved but there's no error message](https://trello.com/c/94UhxS14/7957-qa-works-imported-from-bibtex-containing-invalid-urls-are-not-saved-but-theres-no-error-message)

File: src/app/core/record-works/record-works.service.ts [hunk 1]
```diff
@@ -206,7 +206,7 @@ export class RecordWorksService {
       )
   }
 
-  save(work: Work, bibtex?: boolean) {
+  save(work: Work, bibtex?: boolean): Observable<any> {
```
Comment (leomendoza123) [inline src/app/core/record-works/record-works.service.ts:209]: Please define a response object. 
Maybe a simple generic error object is ok like
```suggestion
  save(work: Work, bibtex?: boolean): Observable<{ errors :string}> {
```

File: src/app/record/components/work-stack-group/modals/work-bibtex-modal/work-bibtex-modal.component.ts [hunk 2]
```diff
@@ -85,6 +86,10 @@ export class WorkBibtexModalComponent implements OnInit, OnDestroy {
                   .subscribe((data) => {
                     data.forEach((work) => {
                       that.worksFromBibtex.push(work)
+                      if (work.errors.length > 0 && !that.isAnInvalidWork) {
+                        that.isAnInvalidWork = true
```
Comment (leomendoza123) [inline src/app/record/components/work-stack-group/modals/work-bibtex-modal/work-bibtex-modal.component.ts:90]: Wondering what happens if the user enters an invalid BibTeX and after that, the user fixes it with a valid BibTeX? 

It seems like we never set the `isAnInvalidWork` back to false 🤔

**Review by leomendoza123 — CHANGES_REQUESTED** [review]
Please check my 2 comments

File: src/app/record/components/work-stack-group/modals/work-bibtex-modal/work-bibtex-modal.component.ts (hunk 2 above)
Comment (DanielPalafox) [reply src/app/record/components/work-stack-group/modals/work-bibtex-modal/work-bibtex-modal.component.ts:90]: Every time you want to import a new BibTeX file you need to click cancel and open the modal again

**Review by leomendoza123 — APPROVED** [review]

**Comment by leomendoza123** [comment]
[[QA] Works imported from BibTeX containing invalid URLs are not saved but there's no error message](https://trello.com/c/94UhxS14/7957-qa-works-imported-from-bibtex-containing-invalid-urls-are-not-saved-but-theres-no-error-message)

**Pull request merged**